token-validation-core/src/main/java/no/nav/security/token/support/core/validation/JwtTokenAnnotationHandler.java

```diff
@@ -19,8 +11,18 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-import static no.nav.security.token.support.core.utils.JwtTokenUtil.contextHasValidToken;
-import static no.nav.security.token.support.core.utils.JwtTokenUtil.getJwtToken;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import no.nav.security.token.support.core.api.Protected;
+import no.nav.security.token.support.core.api.ProtectedWithClaims;
+import no.nav.security.token.support.core.api.Protection;
+import no.nav.security.token.support.core.api.Unprotected;
+import no.nav.security.token.support.core.context.TokenValidationContextHolder;
+import no.nav.security.token.support.core.exceptions.AnnotationRequiredException;
+import no.nav.security.token.support.core.exceptions.JwtTokenInvalidClaimException;
+import no.nav.security.token.support.core.exceptions.JwtTokenMissingException;
+import no.nav.security.token.support.core.jwt.JwtToken;
 
 public class JwtTokenAnnotationHandler {
```

Comment:
General comment: this class is getting a bit big and complicated, we might have to look at some refactoring and maybe introduce some new classes.